Samples/outlook-check-item-categories/README.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Use Outlook Smart Alerts"
+page_type: sample
+urlFragment: outlook-add-in-check-item-categories
+products:
+- office-add-ins
+- office-outlook
+- office
+languages:
+- javascript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: 05/05/2022 10:00:00 AM
+description: "Use Outlook Smart Alerts to verify that required color categories are applied to a new message or appointment before it's sent."
+---
+
+# Use Outlook Smart Alerts
+
+**Applies to**: Outlook on Windows
+
+## Summary
+
+This sample uses Outlook Smart Alerts to verify that required color categories are applied to a new message or appointment before it's sent. Specific keywords detected in the subject or body of the item determine the required categories. If no categories or only some of the required categories are applied to the message or appointment, the add-in blocks the item from being sent and alerts the user to apply the missing categories. The user can apply categories to the item from the add-in task pane.
+
+Smart Alerts and its related events, `OnMessageSend` and `OnAppointmentSend`, are currently available in preview. For documentation related to this sample, see [Use Smart Alerts and the onMessageSend event in your Outlook add-in (preview)](https://docs.microsoft.com/office/dev/add-ins/outlook/smart-alerts-onmessagesend-walkthrough).
```

Comment:
Consider adding a link to the categories article too.

Reply:
Thanks. Added the link.